pyani_plus/public_cli_args.py

```diff
@@ -160,3 +160,18 @@
 OPT_ARG_TYPE_EXECUTOR = Annotated[
     ToolExecutor, typer.Option(help="How should the internal tools be run?")
 ]
+OPT_ARG_TYPE_COVERAGE_EDGES = Annotated[
+    str,
+    typer.Option(
+        help="How to resolve asymmetrical ANI coverage results for edges in the graph (min, max or mean).",
+        rich_help_panel="Method parameters",
+    ),
+]
+
+OPT_ARG_TYPE_IDENTITY_EDGES = Annotated[
+    str,
+    typer.Option(
+        help="How to resolve asymmetrical ANI identity results for edges in the graph (min, max or mean).",
+        rich_help_panel="Method parameters",
+    ),
+]
```

Comment:
_If_ these are only likely to be used for the one public CLI function, perhaps they can live directly in the function definition?